tensorflow_blade/src/pybind.cpp

```diff
@@ -17,7 +17,30 @@
 
 #include "src/tf_compatible_version.h"
 
+#if BLADE_WITH_INTERNAL
```

Comment:
How about wrap all internal bindings into a single header, so as to remove the repeated `#if`s and hie-related macros. Like:
```python
PYBIND11_MODULE(_tf_blade, m) {
  m.doc() = "Utils for tf blade.";
  m.def("compatible_tf_version", &compatible_tf_version);
#if BLADE_WITH_INTERNAL
  blade::internal::pybind(m);
#endif
}
```

Reply:
done